### DISCUSSION TITLE
Ikea Tradfri bulb unsupported

### DISCUSSION THREAD

**Discussion by seintuit** [Koenkk/zigbee2mqtt#19084] [Devices]
Hi,
I just bought an Ikea TRÅDFRI bulb (TRADFRIbulbG125E26WSopal470lm)

https://www.ikea.com/kr/en/p/tradfri-led-bulb-e26-470-lumen-smart-wireless-dimmable-white-spectrum-globe-90441344/
( https://www.zigbee2mqtt.io/devices/LED1936G5.html )

Models of the same shape are sold under different product names in each country.
I tried pairing with Z2M several times, but they all failed.
How can I resolve this ?

-------------------------

warn  2023-09-22 23:01:45: Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
warn  2023-09-22 23:01:48: Received message from unsupported device with Zigbee model 'TRADFRIbulbG125E26WSopal470lm' and manufacturer name 'IKEA of Sweden'
warn  2023-09-22 23:01:48: Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
warn  2023-09-22 23:01:51: Received message from unsupported device with Zigbee model 'TRADFRIbulbG125E26WSopal470lm' and manufacturer name 'IKEA of Sweden'
warn  2023-09-22 23:01:51: Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
warn  2023-09-22 23:01:54: Received message from unsupported device with Zigbee model 'TRADFRIbulbG125E26WSopal470lm' and manufacturer name 'IKEA of Sweden'
warn  2023-09-22 23:01:54: Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

**Comment by Koenkk** [comment]
Added!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)
